src/main/java/solvd/laba/ermakovich/hf/aggregate/AccountAggregateServiceImpl.java

```diff
@@ -20,19 +20,21 @@ public class AccountAggregateServiceImpl implements AccountAggregateService {
     private final AccountRepository accountRepository;
 
     @Override
-    public Mono<Void> apply(EventRoot eventRoot) {
-        if (DeleteAccount.EVENT_TYPE.equals(eventRoot.getEventType())) {
+    public Mono<Void> apply(CreateAccount event) {
+        return accountQueryService.findByIdOrCreate(event.getAggregateId())
+                .flatMap(aggregate -> {
+                    ((Event)event).copyTo(aggregate);
+                    accountRepository.save(aggregate).subscribe();
+                    return Mono.empty();
```

Comment:
do not return `Mono.empty()`

this can lead to disaster while debugging to understand what's going on